src/QueryBuilder.php

```diff
@@ -108,6 +112,17 @@ public function allowedFilters($filters) : self
         return $this;
     }
 
+    public function allowedFields($fields) : self
```

Comment:
Can we add support for `$builder->allowedFields('*')` and enable that by default? 

A bit like `$builder->allowedSorts('*')` - this way this entire feature will be backward compatible!

Reply:
@AlexVanderbist i added support for ```$builder->allowedFields('*')``` but i think by default the method allows all fields if nobody used ```$builder->allowedFields(['column'])```  [check this test ](https://github.com/spatie/laravel-query-builder/blob/master/tests/FieldsTest.php#L25)

Reply:
Thanks, you're right!